src/flat-button.jsx

```diff
@@ -266,13 +266,10 @@ const FlatButton = React.createClass({
       position: 'relative',
       overflow: 'hidden',
       backgroundColor: hovered ? buttonHoverColor : buttonBackgroundColor,
-      lineHeight: buttonHeight + 'px',
+      height: buttonHeight,
```

Comment:
Since we are no longer using the FlatButton. I think that we can remove this change.